### DISCUSSION TITLE
HELP: 'My Tag' Rekordbox with One Tagger?

### DISCUSSION THREAD

**Discussion by junior-fromBrazil** [Marekkon5/onetagger#374] [Q&A]
How to synchronize, or include, One Tagger tags in the 'Rekordbox My Tag' column, which then works in the 'Track Filter' button on CDJs and XDJs.

Can anyone give me this tip, please?

![Captura de Tela 2024-04-09 às 22 26 26](https://github.com/Marekkon5/onetagger/assets/166572736/0ef68d0a-e721-4684-bc3e-e58c53997639)

**Comment by mtsbh** [comment]
+1 
would be great feature

**Comment by Marekkon5** [comment]
Sadly Rekordbox's My Tag is stored in their internal DB. OneTagger cannot read it, it can only read audio file tags. You could potentially achieve it since Rekordbox allows copying the tag data into the comment tag.
